lib/pdk/generators/module.rb

```diff
@@ -68,7 +68,7 @@ def self.invoke(opts = {})
           end
         end
 
-        PDK.answers.update!('template-url' => template_url)
+        PDK.answers.update!('template-url' => template_url) unless template_url == default_template_url
```

Comment:
I don't get the intent of the original line - is this updating the answer for template_url with a local copy of the template instead of the initial remote URL?

Reply:
@james-stocks yeah, the intent is that if a user specifies a custom template URL (for example, a company which has their own internal module template that they'd like to use), it will be remembered for subsequent `pdk new module` runs.

@DavidS good point!